res/values/options.xml

```diff
@@ -337,4 +337,7 @@
 
     <string name="opt_display_realtime_pp_counter_title">Display real-time PP counter</string>
     <string name="opt_display_realtime_pp_counter_summary">Displays a real-time PP counter during gameplay. May increase loading time significantly, especially in long beatmaps</string>
+
+    <string name="opt_enable_video_title">Enable video</string>
+    <string name="opt_enable_video_summary">Enable video playback during gameplay (Android 6 or newer)</string>
```

Comment:
If we're going to support non-1x speed multiplier video playback in devices below Android 6, this description is inaccurate.